.ruby-version

```diff
@@ -1 +1 @@
-3.2.0
+3.1.2
```

Comment:
Can we use latest versions  ? If you have problems installing, checkout https://dev.trk.in.rs/2023/01/11/install-ruby-using-brew-rbenv.html#installing-ruby-using-rbenv